addons/mail/static/src/models/chat_window.js

```diff
@@ -342,6 +342,16 @@ registerModel({
         _computeHasCallButtons() {
             return Boolean(this.thread) && this.thread.rtcSessions.length === 0 && ['channel', 'chat', 'group'].includes(this.thread.channel_type);
         },
+        /**
+         * @private
+         * @returns {boolean|FieldCommand}
+         */
+        _computeHasCloseAsBackButton() {
+            if (this.isVisible && this.messaging.device.isMobile) {
+                return true;
+            }
+            return clear();
```

Comment:
I slightly dislike the `return clear()` here. I don't get the point of doing that, it just looks more opaque and less straightforward.

Reply:
`clear()` is important to tell that no branching decision defaults.
It's clearly not the same as returning `false`, which is a strong decision, and goes against what is a (IMO, good) `default`.